DataFormats/Scouting/interface/Run3ScoutingParticle.h

```diff
@@ -1,17 +1,65 @@
-#ifndef DataFormats_Run3ScoutingParticle_h
-#define DataFormats_Run3ScoutingParticle_h
+#ifndef DataFormats_Scouting_Run3ScoutingParticle_h
+#define DataFormats_Scouting_Run3ScoutingParticle_h
 
 #include <vector>
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```
See https://github.com/cms-sw/cmssw/pull/36840#issuecomment-1028044479. :)

Reply:
Sorry... missed that line too. One sec and I'll fix it.